src/sentry/static/sentry/app/views/stream/stream.jsx

```diff
@@ -637,6 +638,14 @@ const Stream = createReactClass({
   },
 
   renderGroupNodes(ids, statsPeriod) {
+    // Restrict this guide to only show on a project with less than two issues
+    // because anchors light up all issues on the page. Also restrict to new users (joined<30 days)
```

Comment:
<= 2 issues is for newness, not because it lights up all the issues on the page?